app/controllers/barbeque/sns_subscriptions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Barbeque::SnsSubscriptionsController < Barbeque::ApplicationController
+  def index
+    @sns_subscriptions = Barbeque::SNSSubscription.all
```

Comment:
Haha, I noticed that I didn't implement pagination for many resources in Barbeque. (sorry)